nservicebus/messaging/publish-subscribe/index_disable-publishing_core_[7.2,).partial.md

```diff
@@ -0,0 +1,5 @@
+##### Disabling publishing
+
+By default, NServiceBus requires a subscription persister to be configured. If an endpoint does not publish any events, the publish functionality can be disabled to avoid providing a subscription persister:
```

Comment:
```suggestion
By default, NServiceBus requires a subscription persister to be configured. If an endpoint does not have access to any supported data store and does not publish any events, the publishing infrastructure can be disabled:
```

Reply:
I wanted to avoid that statement given that it enables you to not provide a storage in the first place instead of only opting for that in case your storage doesn't support it. In case you don't use sagas, it becomes less and less common to have a storage in the first place?